### PR TITLE
feature/digital-ocean-1-click-installer

### PR DESCRIPTION
**Pull request by dabeeeenster**
App Spec to deploy to digital ocean

Also updates docker to run gunicorn without touching `/tmp` as it can sometimes be read only

one click URL is 
https://cloud.digitalocean.com/apps/new?repo=https://github.com/flagsmith/flagsmith/tree/main

### CODE REVIEW

File: .do/deploy.template.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      repo_clone_url: https://github.com/flagsmith/flagsmith.git
+    envs:
+      - key: DATABASE_URL
+        scope: RUN_TIME
+        value: ${flagsmith-db.DATABASE_URL}
+      - key: DJANGO_ALLOWED_HOSTS
+        scope: RUN_TIME
+        value: '*'
+  databases:
+  - name: flagsmith-db
```
Comment (matthewelwell) [inline .do/deploy.template.yaml:21]: Is this assuming that the user already has a db in their DO environment with this name?

Comment (dabeeeenster) [reply .do/deploy.template.yaml:21]: No - it is created automatically - pretty impressive!

Comment (matthewelwell) [reply .do/deploy.template.yaml:21]: Oh cool, and I guess Postgres is the default then?

**Review by matthewelwell — APPROVED** [review]

File: .do/deploy.template.yaml (hunk 1 above)
Comment (dabeeeenster) [reply .do/deploy.template.yaml:21]: Yeah its the same for https://render.com/ - databases are just postgres now!

**Pull request merged**